src/main/java/business/controllers/RequestController.java

```diff
@@ -557,11 +560,10 @@ public RequestRepresentation unclaim(
             @PathVariable String id,
             @RequestBody RequestRepresentation request) {
         log.info("PUT /requests/" + id + "/unclaim");
-        HistoricProcessInstance instance = requestService.getProcessInstance(id);
```

Comment:
I think the reason that fetching the process instance was the first action, is that it throws an error if it does not exist, before trying to get the current task. Then the error is "Process not found" instead of "Task not found".

Reply:
ahh, I see! I thought it was maybe some copy-paste (based on claim implementation) issue